source/base/DateTime.ooc

```diff
@@ -41,6 +41,7 @@ DateTime: cover {
 	_ticks: ULong = 0
 	ticks ::= this _ticks
 	init: func@ (=_ticks)
+	init: func@ ~fromNanoseconds (nanoseconds: ULong) { this _ticks = nanoseconds / This nanosecondsPerTick }
```

Comment:
But why stick to ticks internally? Do the extra two zeros from reading nanoseconds instead of ticks confuse anyone?